css-layout-classes/styles.css

```diff
@@ -0,0 +1,57 @@
+.container {
+  max-width: 800px;
+  margin: 1px auto;
+}
+
+.row {
+  display: flex;
+  width: 100%;
+}
+
+.column-link {
+  width: calc(100%/3);
+}
+
+.column-super-smash {
+  width: calc(100%/3);
+}
+
+.column-yoshi {
+  width: calc(100%/3);
+}
```

Comment:
I would consolidate these classes into one `column-third` class since the sizes are all the same.